build.sh

```diff
@@ -4,5 +4,5 @@ rm -rf build
 mkdir -p build
 cd build
 cmake .. -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_BUILD_TYPE=Debug # use Debug for easy debugging with gdb
-make # or ninja, depending on your system
-sudo make install # or sudo ninja install
+make                                                          # or ninja, depending on your system
```

Comment:
nit: to use `ninja` we need to add `-G Ninja` in `cmake` command. @lukhnos would you like to create a small PR for this?

https://github.com/openvanilla/fcitx5-mcbopomofo/issues/96